test/e2e/knativeservingdeployment_test.go

```diff
@@ -83,35 +83,34 @@ func deploymentRecreation(t *testing.T, clients *test.Clients, names test.Resour
 		t.Fatalf("No deployment under the namespace %q was found",
 			test.ServingOperatorNamespace)
 	}
-	// Delete the deployments one by one to see if they will be recreated.
-	for _, deployment := range dpList.Items {
-		if err := clients.KubeClient.Kube.AppsV1().Deployments(deployment.Namespace).Delete(deployment.Name,
-			&metav1.DeleteOptions{}); err != nil {
-			t.Fatalf("Failed to delete deployment %s/%s: %v", deployment.Namespace, deployment.Name, err)
-		}
+	// Pick up the first deployment for recreation test.
```

Comment:
Drop this change from this PR please. There's no reason both would change at once (and thus be rolled back at once).